README.md

```diff
@@ -1,410 +1,61 @@
-| `instance_hdd_size` |                                          | 10                                                                        | Sets the instance hard disk size in GB                                                                                                                                                                                                                                                           |
-| `instance_type`     | `m`, `l`, `xl`                           | `m`                                                                       | Sets the instance CPU size. You can also specify vendor specific machines in AWS i.e. `t2.micro`. [See equivalences](#Supported-vendors) table below.                                                                                                                                            |
-| `instance_gpu`      | ``, `tesla`, `k80`                       | ``                                                                        | Selects the desired GPU for supported `instance_types`.                                                                                                                                                                                                                                          |
-| `ssh_private`       |                                          |                                                                           | An SSH private key in PEM format. If not provided, one private and public key wll be automatically generated and returned in `terraform.tfstate`                                                                                                                                                 |
-
-### Machine
+- Rapidly move local machine learning experiments to a cloud infrustructure
+- Take advantage of training models on spot instances without losing any progress
+- Configure provisioning of compute resources from any of the supported vendors in a unified manner
+- Automatically unregister and remove cloud resources (never forget to turn your GPU off again)
 
-Setup instructions:
+## Prerequisites
 
-1. Setup your provider credentials as ENV variables
+To use Iterative Provider you will need to:
 
-<details>
-<summary>AWS</summary>
-<p>
+- [Install](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) Terraform 1.0+.
+- Make sure you have an account with a cloud vendor of choice, and have the respective authentication credentials saved into environment variables. Check out cloud-specific authentication method details in the [docs](<(https://registry.terraform.io/providers/iterative/iterative/latest)>)
```

Comment:
```suggestion
- Make sure you have an account with a cloud vendor of choice, and have the respective authentication credentials set as environment variables. Check out cloud-specific authentication method details in the [docs](<(https://registry.terraform.io/providers/iterative/iterative/latest)>)
```

Nitpick: the suggestion is far from ideal, but “save as” seems a bit too “graphical” for environment variables.